Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -643,7 +643,7 @@ public class PBXProjGenerator {
                 }
             }
 
-            let embed = dependency.embed ?? (!dependencyTarget.type.isLibrary && (
+            let embed = dependency.embed ?? (!dependencyTarget.type.isLibrary && dependencyTarget.type != .staticFramework && (
                 target.type.isApp
                     || (target.type.isTest && (dependencyTarget.type.isFramework || dependencyTarget.type == .bundle))
             ))
```

Comment:
Over time this truly has become the hardest to read expression in the entire codebase 😂

Would love to get this pulled out into a proper function with multiple lines and comments. 
Maybe even in a switch against the types

```swift
switch(target.type, dependencyTarget.type) {
...
}
```
That's not the responsibility of this PR, but if you wanted to add it that would be great! Would need some good test coverage too